plutus-core/plutus-core/src/PlutusCore/Default/Universe.hs

```diff
@@ -75,7 +74,7 @@ feature and have meta-constructors as built-in functions.
 data DefaultUni a where
     DefaultUniInteger    :: DefaultUni (Esc Integer)
     DefaultUniByteString :: DefaultUni (Esc BS.ByteString)
-    DefaultUniChar       :: DefaultUni (Esc Char)
+    DefaultUniText       :: DefaultUni (Esc Text.Text)
```

Comment:
Considering the comment below, maybe we should also call this `DefaultUniString`

Reply:
```suggestion
    DefaultUniString       :: DefaultUni (Esc Text.Text)
```